### DISCUSSION TITLE
How to drag and drop a component from separate layout this grid-layout?

### DISCUSSION THREAD

**Discussion by sheikalthaf** [katoid/angular-grid-layout#24] [Q&A]
Hi,

I have a list of widgets and i want to drag and drop those widgets into this grid layout. It is possible to do that with the help of angular material drag and drop or any other hack?

**Comment by llorenspujol** [comment]
Hi @sheikalthaf ,

Right now there is no easy way possible to do it… maybe some internal hack could make it work, like adding a gridItem (with a dragging active state) when your widget enters on the grid, and removing it if the widgets gets out of the grid without being dragged... but those would be really hardcore hacks😅 .

This question is also related to several other people asking for dragging grid items between grids, or in this case externally from another source. I would consider this feature as high priority. I have to think about how would it be resolved, like super-builtin solution or just provide the tools to allow it to the people who needs it (maybe preferred since is less development time and would cover more cases...). 
But as you see, it would not be immediately solved.

**Reply by sheikalthaf** [reply]
@llorenspujol thanks for response. I really love to see this feature available. If any help needed I'm ready to help.
